examples/python/hello-world.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+
+import requests
+from flask import Flask, request
+
+logging.basicConfig(level=logging.DEBUG)
+
+APP_PORT = 8080
+APP_HOST = 'localhost'
+ROOT_URL = 'http://localhost:8080'
```

Comment:
Can this be made configurable using environment variables? The port and root URL specifically should be configurable. Please verify it works with an ngrok URL.

Reply:
Noted! will make it configurable and push commit.